### DISCUSSION TITLE
You should always be able to see tiles adjacent to those you own

### DISCUSSION THREAD

**Discussion by kalekundert** [kxgames/seacow_spreadsheet_prototype#13] [Ideas]
This nuance occurred to me while I was thinking about how to handle tile expansion/exploration today, and I wanted to write it down so I don't forget.

First of all, I can't remember if we actually decided on this or not, but I was imagining that exploring a tile would only reveal to you the state of the tile on the turn you explored it; you wouldn't get any future updates about that tile unless you explored it again.  In other words, you won't necessarily see where your opponent has expanded unless you keep scouting.  (There are a bunch of variants we could do on this fog-of-war theme, too.  Maybe exploring a tile gives you up-to-date info for a few turns, maybe you could pay more to stay updated for more turns, maybe you get longer/permanent updates after you explore a tile enough times, etc.)

Anyways, if you want to pay to expand into a tile, the game will have to check whether your opponent controls that tile first.  If they do, the game obviously can't let you expand there.  But you may not actually be able to see that opponent, if your scouting is out of date.  This would cause a few annoying issues, namely that (i) it sucks to have the game veto something you're trying to build and (ii) it gives players a (limited) way to learn where their opponent is without having to actually pay to scout.

The only way to avoid this issue is to give the player complete vision over any tiles that they could expand into, which is presumably any tiles adjacent to those they already control.  (This still leaves the possibility that two players could try to claim the same tile on the same turn.  The prototype will have to handle that somehow, but it won't be as much of an issue for the real-time game.)

**Comment by alexmitchell** [comment]
I agree that having the game veto an action would kinda suck. I don't think the second issue would be a problem in the prototype and might be avoidable in the full game depending on other game mechanics. Maybe... But I suppose this is just splitting hairs and addressing both of them now wouldn't be a big deal.

I see an alternative solution: Players can see the ownership of tiles adjacent to their own. The player can't see what's in it until they explore it, but they at least know they can't expand there.

As for both players expanding to the same tile on the same turn, the actions could negate each other and both players can try to buy the tile again. Kinda like a bidding war: whoever has the perseverance (and matching funds) gets the tile. Also it would be easy to implement.

**Comment by kalekundert** [comment]
I actually really like the bidding war idea.  And it's not hard to see how we could do something like that in the real game, either: just have it take a while to "buy" a tile, and give other players to opportunity to make higher bids during that time.  Bidding wars are a great mechanic for strategy games!
